### ISSUE TITLE
iPad Air 2 wont rejailbreak

### ISSUE THREAD

**Issue by rsidhuu** [kpwn/yalu102#314]
I tried to install Filza and Activator as my first two tweaks and doing so seemed okay. I then clicked respring and it went into safe mode. It automatically booted out of safe mode into no substrate mode. I then rebooted myself to normal no jailbreak and when I try to do it now it will either just reboot me or tell me it failed. Tried 15 or so times.

**Comment by Mila432** [comment]
Try more

**Comment by rsidhuu** [comment]
I have been trying since I posted it but in fear of being stuck like this after the iOS 10.2 window closes I decided to just restore. Just posted here to let everyone know but I guess it will be closed now as I don't have the issue anymore. @Mila432

**Comment by tangalbert919** [comment]
Isn't the iPad Air 2 still a work-in-progress? We got the offsets, but the memory bit thing is still...in the works.
Can't find a better way to explain it.

**Comment by tangalbert919** [comment]
rsidhuu, save your SHSH blobs right now before that signing window ends. There's instructions on Reddit, and on Google. Google searches everywhere (except for the Dark Web, but why would people go there?).

**Comment by rsidhuu** [comment]
@tangalbert919 I see. I just got home and saw it was added in beta 7 and as I don't really care for this device much I thought I try it out. Better this than the iPhone I mean.

**Comment by rsidhuu** [comment]
@tangalbert919 I have already. Would rather easily restore in iTunes and wait than use futurerestore

**Comment by mody199610** [comment]
@rsidhuu , It still needs some work but however after some tries it would successfully jailbreak  , Please be patient since this is still a beta and requires more coding from @kpwn

**Comment by rsidhuu** [comment]
@mody199610 yes totally. I just decided to check it out for fun. My iPhone is on 10.2 and I won't play around with that, but this is a spare so I said why not. Will be waiting for final now though.

**Comment by 1337jay** [comment]
It will eventually work, I restarted my iPad and tried dunno, probably near 50 times but it finally worked (iPad Air 2, 10.2).

**Comment by zencowboy007** [comment]
Doing it still (about 14 times) and continuing on iPad Air 2 10.2).  Just at moment cannot seem to get the cydia icon/app on each reboot thru the yalu 7 beta yet  :-(

**Comment by kaidok19** [comment]
The best luck I've had is holding on "go" for few seconds and usually works within 4 to 5 tries

**Comment by zencowboy007** [comment]
Worked Kaidok19!!!! Just held down on the Go icon for about 10 seconds then let up and it took about 20-30 to finally do the working icon and rebooted and have cydia app:-)

**Comment by kaidok19** [comment]
Glad you got it, sometimes it takes a little longer but this way is quickest I've found !=:(

**Comment by Dibbyo456** [comment]
having the the same problem for me.

**Comment by kaidok19** [comment]
Yeah it's finicky bro it will work just keep trying, and then try not to reboot.

**Comment by Dibbyo456** [comment]
@kaidok19 worked! after 15 retry.

**Comment by igobythisname** [comment]
Thank you guys for all your feedback! I'm new here, having a similar issue with my iPad Air 2 on iOS 10.2. I purposefully rebooted to ensure I could re-jailbreak and immediately noticed I was getting 'failed, retry' message, and consecutive restarts with no jailbreak success. I exhausted my efforts trying to close apps in the switcher, Cydia being one of them, but each time I hit 'go' in yalu102 the iPad just restarts and the same apps are still in the switcher. So what I did next, I forced closed all the apps, then I manually powered OFF the iPad using the Sleep/Power button, then powered back up, and did as @zencowboy007 mentioned and held 'go' for 10 seconds, then FINALLY it resprang and I saw it was jailbroken as my Cloaky custom status bar appeared! This is big ol' d00zie of a bug! Does @qwertyoruiopz check in on here or nah? Thanks again, guys!
